next.config.js

```diff
@@ -1,11 +1,10 @@
 /** @type {import('next').NextConfig} */
+
+const { i18n } = require("./next-i18next.config")
+
 const nextConfig = {
 	reactStrictMode: true,
 	swcMinify: true,
```

Comment:
The `i18n` property is missing now.
The config object should look like this:
```
const nextConfig = {
	reactStrictMode: true,
	swcMinify: true,
	i18n
}
```
Please add the missing property.

Reply:
Whoops, sorry about that 😂
I even took the time to run it locally and didn't see an error 🤦🏻‍♂️

I'll fix it right now.
Thanks